R/msg.R

```diff
@@ -23,8 +23,13 @@ msg = function(fig = "3.6", show_code = TRUE, print_plot = TRUE, filter = 0){
     fig = graphnr$graph[graphnr$nr == fig]
   }
   rfile = system.file("examples", paste0(fig, ".R"), package = "MSG")
-  if (print_plot) source(rfile, encoding = "UTF-8")
+  if (print_plot) {
+    opar = par(no.readonly = TRUE)
+    source(rfile, encoding = "UTF-8")
+    par(opar)
```

Comment:
用 `on.exit()` 保证重置参数一定会成功，否则要是 `source()` 出错的话，下一行 `par(opar)` 就不能执行了。

```suggestion
    on.exit(par(opar), add = TRUE)
    source(rfile, encoding = "UTF-8")
```

Reply:
学习了，谢谢！代码已进一步完善，请查阅。